MODULE.bazel

```diff
@@ -0,0 +1,21 @@
+module(
+    name = "xacro",
+    version = "2.0.11",
+)
+
+bazel_dep(name = "rules_license", version = "1.0.0")
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "rules_python", version = "0.40.0")
+
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
+python.toolchain(
+    python_version = "3.11",
+)
+
+pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
+pip.parse(
+    hub_name = "pypi",
+    python_version = "3.11",
+    requirements_lock = "//bazel:requirements_lock.txt",
+)
+use_repo(pip, "pypi")
```

Comment:
Consider adding support for multiple Python versions: https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/grpc/1.68.0/MODULE.bazel#L49-L78.

Reply:
Done, thanks.